src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -1079,6 +1081,24 @@ public static String prefixName(String name) {
     return PDAO_PREFIX + name;
   }
 
+  private Schema buildRowMetadataSchema() {
+    List<Field> fieldList = new ArrayList<>();
+    fieldList.add(Field.of(PDAO_ROW_ID_COLUMN, LegacySQLTypeName.STRING));
```

Comment:
Is there a reason why we use `LegacySQLTypeName.STRING` instead of `StandardSQLTypeName.STRING`?

Reply:
We use LegacySQLTypes for all of our BigQuery table columns. I think we could migrate to standard SQL, but I'm not sure it makes sense to switch for just this one table. I can create a ticket for us to migrate.

Reply:
Ticket: https://broadworkbench.atlassian.net/browse/DR-2271

Reply:
I don't think there's any difference in how the two different types work; they're both converted into `LegacySQLTypeName` internally in the `Field` constructor code. See `Field.Builder.setType(StandardSQLTypeName, Field...)`